src/advanced/node-management.md

```diff
@@ -55,105 +55,85 @@ or IPv6 address formatted as 4 octets separated by dots (for example,
-  The public key of the peer or `null` if the public key is unknown
+  Public key of the peer or `null` if the public key is unknown.
 - **state**: OutgoingConnectionState  
-  Current connection state
+  Current connection state.
 
 ### ServiceInfo
 
 `ServiceInfo` is a JSON object with the following fields:
 
 - **id**: integer  
-  Unique service identifier
+  Unique service identifier.
 - **name**: string  
-  Unique string service identifier
+  Unique string service identifier.
 
 ### BlockHeader
 
 `BlockHeader` is a JSON object with the following fields:
 
 - **height**: integer  
-  The height of the block
+  Height of the block.
 - **prev_hash**: Hash  
-  The hash of the previous block
+  Hash of the previous block.
 - **proposer_id**: integer  
-  ID of the validator that created an approved block proposal
-- **schema_version**: integer  
-  Information schema version. Currently, `0`
+  ID of the validator that created an approved block proposal.
 - **state_hash**: Hash  
   Hash of the current [Exonum state][blockchain-state] after applying
-  transactions in the block
+  transactions in the block.
 - **tx_count**: integer  
-  Number of transactions included into the block
+  Number of transactions included into the block.
 - **tx_hash**: Hash  
-  The root hash of the transactions Merkle tree
+  Root hash of the transactions Merkle tree.
 
 ### Time
 
 `Time` is a string that combined date and time in UTC as per [ISO 8601][ISO8601]
 (for example, `2018-05-17T10:45:56.057753Z`).
 
-### Precommit
-
-`Precommit` is a message, serialized according to
-[message serialization rules](../architecture/serialization.md#message-serialization).
-
-- **body**: Object  
-  The content of the `Precommit` message
-- **body.block_hash**: Hash  
-  The hash of the current block (the `Precommit` message was created for)
-- **body.height**: integer  
-  The height of the current block
-- **body.propose_hash**: Hash  
-  Hash of the corresponding Propose
-- **body.round**: integer  
-  The round when the block proposal was created
-- **body.time**: Time  
-  UTC time of the validator that created the block proposal
-- **body.validator**: integer  
-  ID of the validator that created this `Precommit` message
-- **message_id**: integer  
-  ID of the `Precommit` message. Equals `4`
-- **protocol_version**: integer  
-  The major version of the Exonum serialization protocol. Currently, `0`
-- **service_id**: integer  
-  Unique service identifier. Equals `0`
-- **signature**: Signature  
-  `Precommit` message creator's signature
-
 ### SerializedTransaction
 
-`SerializedTransaction` is a JSON object corresponding to
-[transaction serialization format](../architecture/serialization.md#message-serialization).
+`SerializedTransaction` is an array of bytes in the Protobuf
+[serialization format](../architecture/serialization.md#message-serialization).
+
+### Content
+
+`Content` is a JSON object with the following fields:
+
+- **debug**: a JSON object
```

Comment:
Please add two spaces at the end of the line to introduce a line break here and in similar situations. I guess, these fragments could be formatted differently, e.g.:

- **debug:** Object. Transaction in the deserialized format.

...But then you would need to change the other descriptions in the same way.